app/views/contexts/hub_context.py

```diff
@@ -127,28 +129,38 @@ def get_section_url(section_id, list_item_id) -> str:
 
         return url_for("questionnaire.get_section", section_id=section_id)
 
-    def _get_row_for_repeating_section(self, section_id, list_item_id):
+    def _get_row_for_repeating_section(self, section_id, list_item_id, list_item_index):
         repeating_title = self._schema.get_repeating_title_for_section(section_id)
 
         title = self._placeholder_renderer.render_placeholder(
             repeating_title, list_item_id
         )
 
-        return self._get_row_for_section(title, section_id, list_item_id)
+        return self._get_row_for_section(
+            title, section_id, list_item_index, list_item_id
+        )
+
+    def _get_row_for_section(
+        self, section_title, section_id, list_item_index=None, list_item_id=None
```

Comment:
The new variable should be added to the end of the argument list.
```suggestion
        self, section_title, section_id, list_item_id=None, list_item_index=None
```